### DISCUSSION TITLE
Do not include savings account to net worth.

### DISCUSSION THREAD

**Discussion by vggscqq** [ananthakumaran/paisa#169] [Q&A]
Hi all, I have a savings account and a goal witch connected to it.
I don't want my piggy bank account to be included to net worth nor I want it to be a loan. How should I configure it?

**Comment by ananthakumaran** [comment]
`Networth = Assets - Liabilities`, given savings account is an asset, it will show up there. Could you expand more on why you want Networth to not include a savings account?

**Reply by vggscqq** [reply]
I would like to use my net worth as a "can spend" widget.

**Reply by ananthakumaran** [reply]
Would creating a savings goal achieve the same effect? You can select the list of accounts you want to attach to a goal and the current value is shown in the dashboard.

Networth is not configurable as of now. It just computes the equation I shared above.

**Reply by vggscqq** [reply]
I'm already set the savings goal. Let say I have on my savings account 1.000 and on savings account 500.
I can spend only 1.000, but my net worth is 1.500. Is there a way to maybe create a new widget for only spendable money?

**Reply by ananthakumaran** [reply]
It's not possible to create a widget as of now. What about going to balance page https://demo.paisa.fyi/assets/balance

Another option is to use `Assets:Checking:Bank1`, `Assets:Checking:Bank2`, the balance of these accounts will show up on dashboard page

![image](https://github.com/ananthakumaran/paisa/assets/149238/36434fad-465d-4fe4-9a84-30e4a561e76d)

**Reply by vggscqq** [reply]
Maybe I can change type of account from savings to something that is not included to net worth?

**Reply by ananthakumaran** [reply]
You can read more about accounts [here](https://paisa.fyi/reference/accounts/), but I am afraid what you ask might not fit in anywhere.
